Audit/Program.cs

```diff
@@ -126,11 +157,29 @@ static void RegisterServices(WebApplicationBuilder builder)
         c.IncludeXmlComments(xmlPath);
     });
 
+    //Add logging redaction
+    builder.Logging.EnableRedaction();
+    builder.Services.AddRedaction(x => {
+
+        x.SetRedactor<StarRedactor>(new DataClassificationSet(DataTaxonomy.SensitiveData));
+
+        var hmacKey = builder.Configuration.GetValue<string>("Logging:HmacKey");        
+        if (!string.IsNullOrEmpty(hmacKey))
+        {           
+            x.SetHmacRedactor(opts => {
+                opts.Key = Convert.ToBase64String(Encoding.UTF8.GetBytes(hmacKey));
+                opts.KeyId = 808;
```

Comment:
Maybe not worth it now or what the standard practice is for this, but may want to enumerate or keep a dictionary of the id's if we decide to use this cross service

Reply:
Yeah I agree we should standardize it. Maybe we can look into if there is any best practice for the Ids, but I believe it is just a design decision for our team.